qemu/tests/driver_in_use.py

```diff
@@ -0,0 +1,125 @@
+import re
+import time
+import logging
+from virttest import utils_misc, utils_test, error_context
+from autotest.client.shared import utils
+from avocado.core import exceptions
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Driver in use test:
+    1) boot guest with the device.
+    2) enable and check driver verifier in guest.
+    3) run subtest before / during / after background stress test.
+    4) clear the driver verifier.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def check_bg_status(session, target):
+        """
+        Check the backgroud test status in guest.
```

Comment:
It checks the stress status in guest here.

the target is the process running in the guest.